### ISSUE TITLE
Second example missing something

### ISSUE THREAD

**Issue by cryptoquick** [pshev/preact-fela-simple#1]
In this section: https://github.com/pshev/preact-fela-simple#accessing-props

Should it be this:

```jsx
import { h } from 'preact'
import {style} from 'preact-fela-simple'
 
const Button = ({styles}) => 
    <button className={styles.button({primary: true})}>
        Hello world
    </button>
 
export default style({
  button: props => ({
    fontSize: props.primary ? '15px' : '12px',
    padding: '10px',
    color: 'red'
  })
})(Button)
```

Maybe? I haven't dug in enough to know yet, but I'm just not sure where primary is coming from.

I definitely like the pattern over stock preact-fela, though. You have my support!

**Comment by cryptoquick** [comment]
Wait a second, I think I get it. `props` is the props passed to the component instance? Such as this:
```jsx
<Button primary={true} />
```
Then, is props also passed into the SFC, with the styles property tacked on?

**Comment by cryptoquick** [comment]
Just figured it out. It stems from my use of preact with the `h()` method. I'm not using a JSX transpiler, which I'll admit, I did not detail earlier. So, I'm to use it just like I would any other stateless component, but I need to wrap style with the `h()` method. Then, in the stateless component, I get the props like I normally would. I'm pretty sure that's how it works... Regardless, I've refactored my app to use this, and it's working well so far!
